packages/medusa-react/src/hooks/admin/discounts/mutations.ts

```diff
@@ -3,16 +3,42 @@ import {
   AdminDiscountsRes,
   AdminPostDiscountsDiscountConditions,
   AdminPostDiscountsDiscountConditionsCondition,
+  AdminPostDiscountsDiscountConditionsConditionBatchParams,
+  AdminPostDiscountsDiscountConditionsConditionBatchReq,
   AdminPostDiscountsDiscountDynamicCodesReq,
   AdminPostDiscountsDiscountReq,
-  AdminPostDiscountsReq
+  AdminPostDiscountsReq,
 } from "@medusajs/medusa"
 import { Response } from "@medusajs/medusa-js"
 import { useMutation, UseMutationOptions, useQueryClient } from "react-query"
-import { useMedusa } from "../../../contexts/medusa"
+import { useMedusa } from "../../../contexts"
 import { buildOptions } from "../../utils/buildOptions"
 import { adminDiscountKeys } from "./queries"
 
+export const useAdminAddDiscountConditionItemBatch = (
```

Comment:
todo(if-minor): Rename to be consistent with Seb's comment
```suggestion
export const useAdminAddDiscountConditionResourceBatch = (
```